src/rpc/handlers/Tx.cpp

```diff
@@ -36,6 +36,16 @@ doTx(Context const& context)
     if (!range)
         return Status{Error::rpcNOT_READY};
 
+    if ((request.contains(JS(min_ledger))) &&
+        (request.contains(JS(max_ledger))))
+    {
+        auto& minLedger = request.at(JS(min_ledger));
```

Comment:
Please check if `auto const&` works for these two and if it does - consider enforcing `const` where possible